torch/nn/modules/distance.py

```diff
@@ -1,6 +1,25 @@
+import torch
+from .module import Module
+
+
+class PairwiseDistance(Module):
+    r"""Computes the distance between two vectors using the p-norm.
+    """
+
+    def __init__(self, p=2):
+        super(PairwiseDistance, self).__init__()
+        assert p % 1 == 0
+        self.norm = p
+
+    def forward(self, x1, x2):
+        assert x1.size() == x2.size()
+        eps = 1e-4 / x1.size(1)
```

Comment:
I think it should converge, as it is the value that is added in the [`torch` version](https://github.com/torch/nn/blob/master/PairwiseDistance.lua#L64)